lib/IRGen/IRGenDebugInfo.cpp

```diff
@@ -791,8 +791,7 @@ class IRGenDebugInfoImpl : public IRGenDebugInfo {
     }
 
     Mangle::ASTMangler Mangler;
-    std::string Result = Mangler.mangleTypeForDebugger(
-        Ty, nullptr);
+    std::string Result = Mangler.mangleTypeForDebugger(Ty, nullptr);
```

Comment:
Accidental whitespace change?

Reply:
There were some irgen changes that I was testing out, seems this got through. Thanks!